internal/transporte/model.go

```diff
@@ -0,0 +1,26 @@
+package transporte
+
+
+type TipoMedio int
+const (
+    AUTOBUS = TipoMedio iota
```

Comment:
```suggestion
    AUTOBUS TipoMedio = iota
```

si no, da error de sintaxis